libraries/exoplayer_rtsp/src/main/java/androidx/media3/exoplayer/rtsp/RtspMediaTrack.java

```diff
@@ -56,6 +56,10 @@
 
   private static final String GENERIC_CONTROL_ATTR = "*";
 
+  /** Default width and height for VP8. */
```

Comment:
RFC doesn't mandate any codec specific data(like width, height) to be present in fmtp attributes for VP8.    
RFC also doesn't mentioned any default resolution. So, i use 320X240 as default since it is the default resolution used by Android Software decoder. 
Adding the link for your reference: 
https://cs.android.com/android/platform/superproject/+/master:frameworks/av/media/codec2/components/vpx/C2SoftVpxDec.cpp;drc=749a74cc3e081c16ea0e8c530953d0a247177867;l=70